### PR TITLE
Apps

### PR DESCRIPTION
**Pull request by AlicjaSzu**
**PR intended to be tested with API branch:** feature/extending_app_support

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Translated strings are extracted.
1. [x] Number of API calls is optimized.
1. [x] The changes are tested.
1. [x] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://apps.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by krzysztofwolski** [comment]
@AlicjaSzu our docs will need update. Please contact @korycins for details.

File: src/apps/queries.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { webhooksFragment } from "@saleor/fragments/webhooks";
+import makeQuery from "@saleor/hooks/makeQuery";
+import gql from "graphql-tag";
+
+import { App, AppVariables } from "./types/App";
+import { AppsInstallations } from "./types/AppsInstallations";
+import { AppsList, AppsListVariables } from "./types/AppsList";
+
+const appsList = gql`
+  ${appFragment}
```
Comment (krzysztofwolski) [inline src/apps/queries.ts:11]: Do we need all Apps fields on list view? Metadata, tokens, etc

File: src/apps/components/AppInstallPage/AppInstallPage.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                </ul>
+              )}
+              <Hr className={classes.installSpacer} />
+
+              <Typography
+                variant="body2"
+                className={classes.installPrivacyText}
+              >
+                <FormattedMessage
+                  defaultMessage="Uninstalling the app will remove all your customer’s personal data stored by {name}. After x hours, request will be sent to {name} to erase all of the data. "
```
[the remark this answers is not in the document]
Comment (AlicjaSzu) [reply src/apps/components/AppInstallPage/AppInstallPage.tsx]: i removed this text

File: schema.graphql [hunk 3]
```diff
@@ -4,28 +4,40 @@ schema {
 }
 
 type AccountAddressCreate {
-  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  errors: [Error!]!
+    @deprecated(
```
Comment (dominik-zeglen) [inline schema.graphql:8]: Oops looks like someone used prettier on this

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Handle task failure and fix task duplication - #588 by @dominik-zeglen
 - Move fragments to separate directory to avoid circular imports - #592 by @dominik-zeglen
 - Add order invoices management - #570 by @orzechdev
+- create Apps - #599 by @AlicjaSzu
```
Comment (dominik-zeglen) [inline CHANGELOG.md:14]: How about this?

```suggestion
- Add support for apps - #599 by @AlicjaSzu
```

File: src/apps/components/AppDetailsPage/AppDetailsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+import Container from "@saleor/components/Container";
+import ExternalLink from "@saleor/components/ExternalLink";
+import PageHeader from "@saleor/components/PageHeader";
+import Skeleton from "@saleor/components/Skeleton";
+import { sectionNames } from "@saleor/intl";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import ReactMarkdown from "react-markdown";
+
+import activateIcon from "../../../../assets/images/activate-icon.svg";
```
Comment (dominik-zeglen) [inline src/apps/components/AppDetailsPage/AppDetailsPage.tsx:17]: There's an alias for this

```suggestion
import activateIcon from "@assets/images/activate-icon.svg";
```

**Pull request merged**